src/detect-engine-address.c

```diff
@@ -894,10 +901,8 @@ static int DetectAddressParse2(const DetectEngineCtx *de_ctx,
                         goto error;
                 }
 
-
                 if (DetectAddressParse2(de_ctx, gh, ghn, temp_rule_var_address,
-                                    (negate + n_set) % 2, var_list, recur) < 0)
-                {
```

Comment:
Can we minimize the changes to the functional ones ?

Reply:
I'll check ... i think this was a clang-format change.